utils/json/nullable/enum.go

```diff
@@ -0,0 +1,34 @@
+package nullable
+
+import "strconv"
+
+// EnumNull is the value used to represent JSON null.
+// It should never be used as a value, as it won't get serialized as such.
+const EnumNull = -1
+
+// Enum is a nullable version of a uint8.
+// Enum values should only consist of positive values, as negative values are reserved for internal constants, such as
```

Comment:
(((nitpicking))) Can this comment be limited to 80 columns?

Reply:
sure, should I refactor the rest of the code as well to use 80 cols?

Reply:
or is this just for comments?

Reply:
Code is 100 cols, comments is 80 cols is the convention.

Reply:
k